android/AndroidManifest.xml

```diff
@@ -12,6 +12,7 @@ file, You can obtain one at http://mozilla.org/MPL/2.0/.
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
     <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES"/>
+    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE"/>
```

Comment:
Can you explain why more permissions? The sdk mentions we need this only 
```If you are not targeting the Google Play Store (you need it)``

Reply:
I oriented myself at an example they had but we can probably remove that.